x-pack/legacy/plugins/maps/common/constants.js

```diff
@@ -37,6 +37,10 @@ export function createMapPath(id) {
   return `${MAP_BASE_URL}/${id}`;
 }
 
+export const DEFAULT_ZOOM = 1;
```

Comment:
In my setup, I had to increase the zoom value to **`2`** to get your bounding box, so I'd wait for others to confirm this defaults.

![image](https://user-images.githubusercontent.com/188264/73264463-2d2c3c00-41d3-11ea-84e1-c7f9b013d5ce.png)